docs/guides/setup-channel.mdx

```diff
@@ -205,50 +243,50 @@ Alternatively, you can also access quantum processors with REST APIs, enabling y
-    QiskitRuntimeService.save_account(
-        channel="ibm_cloud",
-        token="<IBM Cloud API key>",
-        instance="<IBM Cloud CRN>",
-        name="<account-name>",
-        # Use `overwrite=True` if you're updating your token.
-        overwrite=True,
-    )
-
     # Load saved credentials
-    service = QiskitRuntimeService(name="<account-name>")
+    service = QiskitRuntimeService()
     ```
 
-    * If you save your credentials to disk, you can use `QiskitRuntimeService()` in the future to initialize your account. The `channel` parameter distinguishes between different account types. When initializing the account, IBM Cloud is the default account used if you have saved credentials for both an IBM Quantum Platform and an IBM Cloud account.
-    * If you are saving multiple accounts per channel, consider using the `name` parameter to differentiate them.
-    * Credentials are saved to `$HOME/.qiskit/qiskit-ibm.json`.  Do not manually edit this file.
-    * If you don't save your credentials, you must specify them every time you start a new session.
-
-    <Admonition type="caution">
-        Account credentials are saved in plain text, so only do so if you are using a trusted device.
-    </Admonition>
+      * If you have saved credentials for both an IBM Quantum Platform account and an IBM Cloud account, IBM Cloud is the default account used when you initialize your account, unless you specify `set_as_default=True` in your IBM Quantum Platform account when you use the `save_account()` method.
+      * If you are saving multiple accounts per channel, consider using the `name` parameter to differentiate them.
+      * Credentials are saved to `$HOME/.qiskit/qiskit-ibm.json`.  Do not manually edit this file.
+      * If you don't save your credentials, you must specify them every time you start a new session.
 
- 1. Test your setup.  Ensure that you can connect to the service:
+1. <span id="cloud-untrusted"></span>**Avoid executing code on an external cloud Python environment to minimize security risks.** If you must use an untrusted environment (on, for example, a public computer), change your API token after each use by deleting it on the [API keys page](https://cloud.ibm.com/iam/apikeys) and creating a new one. To initialize the service in this situation, you can use code like the following:
 
-   ```python
+    ```python
     from qiskit_ibm_runtime import QiskitRuntimeService
-
-    service = QiskitRuntimeService()
-  ```
+    
+    service = QiskitRuntimeService(
+      channel="ibm_cloud", 
+      # Delete your token on the API keys page after entering this code:
+      token="<IBM Cloud API key>", 
+      instance="<IBM Cloud CRN>"
+      )
+    ```
+    <Admonition type="caution">
+      When sharing code with others, ensure that your API token is not embedded directly within the Python script. Instead, share the script without the token and provide instructions for securely setting it up.
+    </Admonition>
```

Comment:
Same comment as before - that this should be higher up. I assume you'll just copy&paste. I like the bolded `Protect your API key!` in the other warning.

Reply:
done in 0be84fc58abae1f333ecfaac1043e739a6e659ce